scripts/config_check/options.h

```diff
@@ -0,0 +1 @@
+/* After running the refresh.sh command, this file will contain a copy of the generated options.h file. */
```

Comment:
My concern is this is a tracked file, so after running the script this show as modified. I think it would be better to not track and have it in the `.gitignore`.

Reply:
added to `.gitignore`